### ISSUE TITLE
Feature Request - Read Analog Gauge / Multi-analog gauge / Multi-digital meters

### ISSUE THREAD

**Issue by bkc** [usefulsensors/person_sensor_docs#1]
Hi,

I'm sorry if this is out of scope for this project. 

Would you please consider supporting a training model that can read an analog gauge and/or multi-gauges as shown below?

I'm planning to read this with an ESP32-CAM and PyTorch, but a pre-built module would be much easier.

![image](https://user-images.githubusercontent.com/119869/196775594-0cbaf7fb-a55e-410c-a03a-8d8b09aebe0e.png)

Another gauge I need to read

![image](https://user-images.githubusercontent.com/119869/196776787-fef2850e-0e83-4ee1-8caf-26e8f4302bbf.png)

The pulse counter on the following gauge works, but direct reading would be easier to integrate and more reliable

![image](https://user-images.githubusercontent.com/119869/196775780-d907960c-ccaf-43c0-a697-934e2e1f2f32.png)

Here's another water meter I need to read

![image](https://user-images.githubusercontent.com/119869/196776521-d3d3ee87-c9ca-4c77-9d7c-8b7bada3f774.png)

Here's another future ESP32-CAM target.

![image](https://user-images.githubusercontent.com/119869/196775970-ea2884a6-a1f4-4ada-b83a-23ac38b15b0c.png)

**Comment by mamin27** [comment]
Yes, good appointment to see firmware and then update it to read of following gauges. But the firmware is not open source.

**Comment by bkc** [comment]
I just purchased (4) of these people detectors to see if I can use them in a building heat management system instead of PIR sensors.

I would happily pay more for a version that supported 'generic analog gauge' (even just a single dial).   The output could be a list of register marks the model understood e.g. (0, 1, 2, 3, 4, 5) and the current perceived value (2.2 in my first example).  

If it also did OCR and returned a list of text strings, the same model might work with the digital power meter as well, though maybe that's asking too much.
